shared/gh/js/views/gh.admin-batch-edit-date.js

```diff
@@ -46,7 +46,9 @@ define(['lodash', 'moment', 'gh.core', 'gh.api.config'], function(_, moment, gh,
      */
     var buildBatchDateObject = function() {
         // Get the checked events from the batch edit container
-        var $rows = $('.gh-batch-edit-events-container tr.info:visible:not(".gh-event-deleted")');
+        var $selectedRows = $('.gh-batch-edit-events-container tr.info:visible:not(".gh-event-deleted")');
+        // Filter the rows that are eligible for updating
+        var $rows = getEligibleRows($selectedRows);
```

Comment:
You can ignore this, I caught on with what you're doing further down the file where you mark the non-eligible rows to be able to style them differently.